tools/api-markdown-documenter/src/api-item-transforms/default-implementations/CreateSectionForApiItem.ts

```diff
@@ -76,9 +79,23 @@ export function createSectionForApiItem(
 	// Render alpha/beta notice if applicable
 	const releaseLevel = getEffectiveReleaseLevel(apiItem);
 	if (releaseLevel === ReleaseTag.Alpha) {
-		sections.push(wrapInSection([new ParagraphNode([alphaWarningSpan])]));
+		sections.push(
+			wrapInSection([
+				new MarkdownBlockContentNode({
```

Comment:
Note for reviewers: like what was done with `MarkdownPhrasingContentNode`, this type and its usages will be removed in a future PR once all custom block content nodes have been removed.